src/components/AssessmentView.jsx

```diff
@@ -636,21 +641,31 @@ const AssessmentView = ({
                     {/* <hr /> */}
 
                     {activeWeightageIndex === index && (
-                      <div className="applied-weightage-list-container flex">
+                      <div className="applied-weightage-list-container flex" style={{gap:"10px"}}>
                         {currentAssessment.task_weightages &&
                           currentAssessment.task_weightages.map(
                             (weightage, weightageIndex) => (
                               <div
```

Comment:
add a return , 
define a variable in which you store find value,
use it below in both of the places instead of writing it two times

Reply:
I have solved